pkg/injector/webhook.go

```diff
@@ -510,10 +470,31 @@ func createMutatingWebhook(clientSet kubernetes.Interface, cert certificate.Cert
 	}
 
 	if _, err := clientSet.AdmissionregistrationV1().MutatingWebhookConfigurations().Create(context.Background(), &mwhc, metav1.CreateOptions{}); err != nil {
-		// TODO(#3962): metric might not be scraped before process restart resulting from this error
-		log.Error().Err(err).Str(errcode.Kind, errcode.GetErrCodeWithMetric(errcode.ErrCreatingMutatingWebhook)).
-			Msgf("Error creating MutatingWebhookConfiguration %s", webhookName)
-		return err
+		// Webhook already exists, update the webhook in this scenario
+		if apierrors.IsAlreadyExists(err) {
+			existingMwhc, err := clientSet.AdmissionregistrationV1().MutatingWebhookConfigurations().Get(context.Background(), mwhc.Name, metav1.GetOptions{})
+			if err != nil {
+				log.Error().Err(err).Str(errcode.Kind, errcode.GetErrCodeWithMetric(errcode.ErrUpdatingMutatingWebhook)).
+					Msgf("Error getting MutatingWebhookConfiguration %s", webhookName)
+				return err
+			}
+
+			existingMwhc.Webhooks = mwhc.Webhooks
+			existingMwhc.Labels = mwhc.Labels
+			if _, err = clientSet.AdmissionregistrationV1().MutatingWebhookConfigurations().Update(context.Background(), existingMwhc, metav1.UpdateOptions{}); err != nil {
+				// There might be conflicts when multiple injectors try to update the same resource
```

Comment:
Could you extend this comment to indicate that it is okay for a conflict error to happen here, and that it should not be treated as an error because it implies another control plane successfully updated the resource.